cpp/src/arrow/util/tracing_internal.cc

```diff
@@ -68,15 +112,14 @@ std::unique_ptr<sdktrace::SpanExporter> InitializeExporter() {
   auto maybe_env_var = arrow::internal::GetEnvVar(kTracingBackendEnvVar);
   if (maybe_env_var.ok()) {
     auto env_var = maybe_env_var.ValueOrDie();
-    if (env_var == "otlp_http") {
-#ifdef ARROW_WITH_OPENTELEMETRY
+    if (env_var == "ostream") {
+      return arrow::internal::make_unique<otel::exporter::trace::OStreamSpanExporter>();
+    } else if (env_var == "otlp_http") {
       namespace otlp = opentelemetry::exporter::otlp;
       otlp::OtlpHttpExporterOptions opts;
       return arrow::internal::make_unique<otlp::OtlpHttpExporter>(opts);
-#else
-      ARROW_LOG(WARNING) << "Requested " << kTracingBackendEnvVar << "=" < < < < env_var
-          " but Arrow was not built with ARROW_WITH_OPENTELEMETRY";
-#endif
+    } else if (env_var == "arrow_otlp_stdout") {
```

Comment:
Should there be, similarly, "arrow_oltp_stderr"?

(btw, what does "oltp" mean?)

Reply:
We could add it.

It stands for [OpenTeLemetry Protocol](https://github.com/open-telemetry/opentelemetry-specification/blob/main/specification/protocol/otlp.md)